recipes/python-pdal/bld.bat

```diff
@@ -0,0 +1,8 @@
+%PYTHON% setup.py build
```

Comment:
Instead of the seperate `bld.bat` and `build.sh`, can you install with pip and then run the tests as part of the `tests` section?

https://conda-forge.org/docs/meta.html#use-pip

Reply:
Thanks. I thought there was probably a way to do this and forgot to go back and double check.

Reply:
I'm honestly not all that familiar with Python unit tests and such. I had previously run `python setup.py test` in the build scripts. What would be the recommended way to run those now that we are doing `pip install`? I saw that I could add `source_files`, along with additional test requirements and commands, but I think the tests still need to be built the "old way" first. Am I missing anything?

Reply:
@chambbj I think you can do this by adding the `test` dir to `source_files`, and then just running `python -m unittest test`.